autofit/mapper/prior_model/representative.py

```diff
@@ -0,0 +1,69 @@
+from collections import defaultdict
+
+from ..prior.abstract import Prior
+
+
+def _value_hash(value) -> int:
+    """
+    When hashing values priors are hashed by their attributes, so that priors with the same attributes hash to the same
+    value. This is important for the grouping of priors in the model representation.
+    """
```

Comment:
```suggestion
    """
    When hashing values, priors are hashed based on their attributes, so that priors with the same attributes hash to the same value. This is important for grouping priors in the model representation.
    """
```